### DISCUSSION TITLE
The GLB file postion is not set on face its show on top

### DISCUSSION THREAD

**Discussion by Amitdygon** [sceneview/sceneform-android#308] [Q&A]
.glb file postion is not write its always show on the top how can i set the 3D model into center 
setRecenterMode(RenderableSource.RecenterMode.CENTER)

Here i attach my glb file https://drive.google.com/file/d/1M2OwRKvtu-vfDtLOAbI3wqZb73XxU779/view?usp=sharing

**Comment by grassydragon** [comment]
Hi!
You need to change your model to use our template here: https://github.com/SceneView/sceneform-android/blob/master/samples/augmented-faces/assets-sources/models/fox.blend
The `README.md` file has been updated recently to include more information on how to use the template.

**Reply by kelvinwatson** [reply]
In my case, the model is from a remote source, not a local asset. How would we center the model programmatically without access to RenderableSource.RecenterMode?
